### ISSUE TITLE
Microsofts flags this library as containing BACK DOORs in certain ps1 files!

### ISSUE THREAD

**Issue by Riebe** [Gerenios/AADInternals#91]
Affected files:
- Common Utils
- WBAWeponizer

**Comment by michaelmsonne** [comment]
Yes, not many providers like this PowerShell Module - but it's with the stuff it's doing and so 😎

You can writelist it, as I not think (and what I have seen over the years..) it will be "allowed"/writelisted for the vendors.

**Comment by NestoriSyynimaa** [comment]
After v0.9.6 release, this should not happen with AADInternals module anymore.

https://aadinternals.com/post/aadinternals-endpoints/
